pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/ContextImpl.java

```diff
@@ -160,6 +161,11 @@ public void setCurrentMessageContext(Record<?> record) {
         this.record = record;
     }
 
+    @Override
+    public Object getCurrentMessage() {
+        return ((PulsarRecord<?>) record).getActualMessage();
```

Comment:
this should just be record.getActualMessage?

Reply:
Oh, well, it was ```getActualMessage()``` for PulsarRecord, but in ContextImpl, I think ```getCurrentMessage()``` was proposed to be used instead. I just took the name from what was proposed in the issue,